floor_generator/lib/misc/extension/string_extension.dart

```diff
@@ -55,7 +55,7 @@ extension NullableStringExtension on String? {
     if (this == null) {
       return 'null';
     } else {
-      return "'${escape(this!)}'";
+      return "'${this!.toEscaped()}'";
```

Comment:
I think the entire function body can be simpler: we can use this?.toEscaped() and use null-coalessing to return the literal 'null'